engine/src/main/java/com/redhat/cloud/notifications/templates/models/Environment.java

```diff
@@ -10,19 +10,14 @@ public class Environment {
     @ConfigProperty(name = "env.name", defaultValue = "local-dev")
     String environment;
 
+    @ConfigProperty(name = "env.base.url", defaultValue = "https://console.redhat.com")
```

Comment:
Should we keep the previous default value which was `/` instead of using `https://console.redhat.com`?
```suggestion
    @ConfigProperty(name = "env.base.url", defaultValue = "/")
```

Reply:
Switched back to the previous default value!